zanata-war/src/main/java/org/zanata/util/UrlUtil.java

```diff
@@ -120,24 +125,36 @@ public String versionUrl(String projectSlug, String versionSlug) {
                 + versionSlug;
     }
 
-    public String editorDocumentListUrl(String projectSlug, String versionSlug,
-            LocaleId targetLocaleId, LocaleId sourceLocaleId) {
-        return getContextPath() + "/webtrans/translate?project=" + projectSlug
+    public String editorDocumentListUrl(boolean fullPath, String projectSlug,
+                    String versionSlug, LocaleId targetLocaleId,
+                    LocaleId sourceLocaleId) {
```

Comment:
I think it would be less confusing to add the extra parameter `fullPath` at the end.